packages/dmn-js-shared/test/spec/base/ManagerSpec.js

```diff
@@ -22,6 +23,37 @@ class TestViewer extends Manager {
 
 }
 
+class ReturnWarningsView extends TestView {
```

Comment:
Can you quickly explain why we add these test views here? For better testing of view changes?

Reply:
Yes - the reason is that I wanted to have an easy way to test the #open API. With the existing views that we have, a thorough test is difficult, since I found it very difficult to create all the situations one could potentially encounter (e.g., having an error but also having parsingWarnings).

Using these dummy views, one can very easily mock an of such situations.

Reply:
Nice!